tasks/configuration.yml

```
@@ -52,9 +52,40 @@
     state: absent
   loop:
     - /etc/icinga2/conf.d/services.conf
-    - /etc/icinga2/conf.d/templates.conf
     - /etc/icinga2/conf.d/hosts.conf
 
+# Twilio configuration
+- name: install twilio sms scripts
+  template:
+    src: 'templates/etc/icinga2/scripts/{{ item }}.j2'
+    dest: '/etc/icinga2/scripts/{{ item }}'
+    owner: icinga
+    group: icinga
+    mode: 0755
+    seuser: system_u
+    serole: object_r
+    setype: etc_t
+    selevel: s0
+  loop:
+    - twilio-sms-host-notification.sh
+    - twilio-sms-service-notification.sh
+  when: icinga2_master_twilio_sms_enabled|bool == true
```

```suggestion
  when: icinga2_master_twilio_sms_enabled
```
should work as well? would be simpler...